cypress/integration/engagement_spec.js

```diff
@@ -1,89 +1,226 @@
-describe('Create an Engagement', () => {
-  it('New Engagement', () => {
+/// <reference types="cypress" />
+
+describe('new engagement', () => {
+
+  it('creates a new engagement', () => {
+    cy.login(
+        Cypress.env('SSO_URL'),
+        Cypress.env('SSO_USER'),
+        Cypress.env('SSO_PASSWORD')
+    );
+    cy.server();
+    cy.route({method: 'POST', url: 'engagements'}).as('createEngagement');
+
+    cy.visit('http://localhost:3000');
```

Comment:
We should avoid hard-coded url's. I think in this case we just use cy.visit('/'); and it should take your configured value.

Reply:
Good catch, It's been removed.